src/gnn/gnn_utils.py

```diff
@@ -76,35 +77,31 @@ def train_model(model: Any, dataloaders: Dict,
                 # statistics
                 running_loss[phase] += loss.item()
 
-        if epoch % 100 == 0:
-            print("running_loss:", running_loss, flush=True)
+        if epoch % 1000 == 0:
```

Comment:
Note that this may change the performance of the GNN approach slightly. Instead, let's leave the GNN numbers at 100. Anyway 1000 would print things out way too slowly